tools/do_release.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+# Copyright 2022 The Vitess Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ROOT=$(pwd)
+if [ "$VTROOT" != "" ]; then
+    ROOT=$VTROOT
+fi
+
+if [ "$RELEASE_VERSION" == "" ]; then
+  echo "Set the env var RELEASE_VERSION with the release version"
+  exit 1
+fi
+
+if [ "$DEV_VERSION" == "" ]; then
+  echo "Set the env var DEV_VERSION with the version the dev branch should have after release"
+  exit 1
+fi
+
+if [ "$VTOP_VERSION" == "" ]; then
+  echo "Warning: The VTOP_VERSION env var is not set, the Docker tag of the vitess-operator image will not be changed."
+  echo -n "If you wish to continue anyhow press enter, otherwise CTRL+C to cancel."
+  read line
+fi
+
+if [ "$GODOC_RELEASE_VERSION" == "" ]; then
+  echo "Warning: The GODOC_RELEASE_VERSION env var is not set, no go doc tag will be created."
+  echo -n "If you wish to continue anyhow press enter, otherwise CTRL+C to cancel."
+  read line
+fi
+
+function updateVersionGo () {
+  echo package servenv > $ROOT/go/vt/servenv/version.go
+  echo  >> $ROOT/go/vt/servenv/version.go
+  echo "const versionName = \"$1)\"" >> $ROOT/go/vt/servenv/version.go
+}
+
+function updateJava () {
+  cd $ROOT/java
+  mvn versions:set -DnewVersion=$1
+}
+
+function updateVitessOperatorExample () {
+  vtop_example_files=$(find -E $ROOT/examples/operator -name "*.yaml")
+  sed -i.bak -E "s/vitess\/lite:(.*)/vitess\/lite:$RELEASE_VERSION/g" $vtop_example_files
+  sed -i.bak -E "s/vitess\/lite:(.*)-mysql80/vitess\/lite:$RELEASE_VERSION-mysql80/g" $(find -E $ROOT/examples/operator -name "*.md")
+  if [ "$VTOP_VERSION" != "" ]; then
+  		sed -i.bak -E "s/planetscale\/vitess-operator:(.*)/planetscale\/vitess-operator:$VTOP_VERSION/g" $vtop_example_files
+  fi
+  rm -f $(find -E $ROOT/examples/operator -regex ".*.(md|yaml).bak")
+}
+
+git_status_output=$(git status --porcelain)
+if [ "$git_status_output" == "" ]; then
+  	echo so much clean
+else
+    echo cannot do release with dirty git state
+    exit 1
+fi
+
+# Preparing the release commit
+updateVitessOperatorExample
```

Comment:
nit: it seems inconsistent that you don't pass in RELEASE_VERSION here as an argument.

Reply:
You're right, I changed it via 5a89899